lib/resolvers.js

```diff
@@ -29,3 +30,17 @@ export const mapLinkList = fieldName => doc => {
     return linkItem;
   }).filter(l => l !== undefined);
 };
+
+export const mapScheduleList = fieldName => doc => {
+  const schedule = doc[fieldName];
+
+  if (!schedule) { return []; }
+  return schedule.map((item) => {
```

Comment:
```javascript
export const mapScheduleList = fieldName => doc => {
  const schedule = doc[fieldName];
  if (!schedule) { return []; }
  return schedule.map((item) => {
    const datetime = pathOr('', ['datetime', 'value'], item);
    const text = pathOr('', ['text', 'value'], item);
    if (datetime || text) {
      acc.push({ datetime, text });
    }
    return acc;
  })
};
```

Note this is a really bad example of a reduce as you're mutating the accumulator (javascript sucks). Would more more efficient to use  `forEach`.